src/components/Navbar.tsx

```diff
@@ -1,4 +1,13 @@
-export const Navbar = () => {
+import { NavLink } from 'react-router-dom';
+import cn from 'classnames';
+import { FC } from 'react';
+
+export const NavBar: FC = () => {
+  const isActiveLink = (isActive: boolean) => cn(
```

Comment:
```suggestion
  const getLinkClasses = (isActive: boolean) => cn(
```
The function name doesn't match what it actually does